www/base/src/app/common/directives/changelist/changelist.tpl.jade

```diff
@@ -16,3 +16,4 @@
         ul.list-group
             li.list-group-item(ng-repeat="change in changes")
                 changedetails(change="change")
+                div {{expandChange()}}
```

Comment:
what is the purpose of this line?

Reply:
We should ideally be calling this function from the directive, but when I do so, then ALL of the changes are getting expanded.

Reply:
I think your problem is a race between locationChangeSuccess and change loaded.
You need to add a $scope.$watch in order to watch for change list updates

Reply:
Thank you, I was able to fix the issue using a watcher :)